### PR TITLE
Added in three lines of text.

### PR DESCRIPTION
**Pull request by SmashBroRazz**
* Retained previous text from last edit.
* Added three new lines of text.

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,7 @@
-Here's my edit.
\ No newline at end of file
+Here's my edit.
+
+Any old edit
```
Comment (alh311) [inline test.txt]: Why not a new edit?

Comment (SmashBroRazz) [reply test.txt]: Oh fine, a new one then.

Comment (SmashBroRazz) [reply test.txt]: Updated.

File: test.txt [hunk 2]
```diff
@@ -1 +1,7 @@
-Here's my edit.
\ No newline at end of file
+Here's my edit.
```
Comment (SmashBroRazz) [inline test.txt:1]: A sample comment!

**Review by alh311 — APPROVED** [review]

**Pull request merged**